### ISSUE TITLE
Add static accessibility checks

### ISSUE THREAD

**Issue by nonprofittechy** [SuffolkLITLab/DAYamlChecker#20]
Minimal, guaranteed correct checks, including:

* missing alt text
* incorrect heading levels
*
